src/main/java/org/broadinstitute/dsde/consent/ontology/datause/MatchResultType.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.dsde.consent.ontology.datause;
+
+import java.util.function.Predicate;
+import java.util.regex.MatchResult;
+
+enum MatchResultType implements Predicate<MatchResultType> {
+  APPROVE,
+  DENY,
+  ABSTAIN;
+
+  public static Boolean Approve(MatchResultType x) { return x == APPROVE; }
+
+  public static Boolean Deny(MatchResultType x) {
+    return x == DENY;
+  }
+
+  public static Boolean Abstain(MatchResultType x) { return x == ABSTAIN; }
+
+  @Override
+  public boolean test(MatchResultType matchResultType) {
+    return false;
+  }
```

Comment:
This method looks a little strange. Perhaps a comment on why this is needed would be useful?

Reply:
I was able to remove a few things, this was able to go! thanks for pointing it out!